modules/oe_theme_helper/src/TwigExtension/TwigExtension.php

```diff
@@ -55,6 +57,7 @@ public function getFilters(): array {
   public function getFunctions(): array {
     return [
       new \Twig_SimpleFunction('to_ecl_icon', [$this, 'toEclIcon'], ['needs_context' => TRUE]),
+      new \Twig_SimpleFunction('trim_textfield', [$this, 'trimTextfield'], ['needs_context' => TRUE]),
```

Comment:
Let's not use needs_context here as we don't really need it, we just need to pass the length as a parameter.

Also let's call it `smart_trim` as it's really the implementation of that behavior.